addons/survey/views/assets.xml

```diff
@@ -58,6 +58,7 @@
             <script type="text/javascript" src="/survey/static/tests/tours/certification_success.js"></script>
             <script type="text/javascript" src="/survey/static/tests/tours/survey.js"></script>
             <script type="text/javascript" src="/survey/static/tests/tours/survey_prefill.js"></script>
+            <script type="text/javascript" src="/survey/static/tests/tours/survey_with_triggers.js"></script>
```

Comment:
``survey_tour_chained_conditional_questions`` ?

Reply:
Agreed for `X_chained_conditional_questions`.

It seems however that usage for the other test tours is to not have "tour" in filename nor tour name (see test_01 to 06 here and files in the assets), but only in the module name. Except for the "session-specific" ones.